connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/DistributedConfig.java

```diff
@@ -67,6 +67,20 @@
             " fails to catch up within worker.sync.timeout.ms, leave the Connect cluster for this long before rejoining.";
     public static final int WORKER_UNSYNC_BACKOFF_MS_DEFAULT = 5 * 60 * 1000;
 
+    /**
+     * <code>offset.storage.topic</code>
+     */
+    public static final String OFFSET_STORAGE_TOPIC_CONFIG = "offset.storage.topic";
```

Comment:
@ewencp 
Yes, that would be good;
Currently, it is passing config's originals (Map<String, ?>)  to OffsetBackingStore and KafkaConfigStorage, would it  be better to pass StandaloneConfig or DistributedConfig to OffsetBackingStore and KafkaConfigStorage dirrectly?
For example, take the config as an parameter of the constructor (just like KafkaConsumer and KafkaProducer); by this way, we can get the value of the config by method of AbstractConfig, like (getShort(), getInt(), getLong(), getString() .......)